src/pages/index.js

```diff
@@ -5,56 +5,73 @@ import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import UserInfo from "../components/UserInfo.js";
 import Section from "../components/Section.js";
+import Api from "../components/Api.js"
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 import {
   initialCards,
+  avatarForm,
   config,
   variable,
   cardForm,
   profileEditModal
 } from "../utils/constants.js";
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "a5172d76-7206-4bb6-b609-2e8e17b83724",
+    "Content-Type": "application/json",
+  },
+});
+
 function handleImageClick(cardData) {
   popupWithImage.open(cardData);
 }
 
-function createCard(item) {
-  const cardElement = new Card(item, "#card-template", handleImageClick);
-  return cardElement.getView()
+function renderCard(item) {
+  const card = new Card(item, "#card-template", handleImageClick, handleDeleteButton, handleLikeClick);
+  return card.getView()
 }
 
-function renderCard(cardData) {
-  const card = createCard(cardData);
-  section.addItem(card);
-}
 
-function handleAddCardFormSubmit(inputValues) {
-  renderCard(inputValues);
-  variable.addCardForm.reset();
-  cardFormFormValidator.changeButtonState();
-  cardPopup.close();
-}
+
 
 //FormValidator
-const profileEditModalFormValidator = new FormValidator(profileEditModal,config);
+const profileEditModalFormValidator = new FormValidator(profileEditModal, config);
 const cardFormFormValidator = new FormValidator(cardForm, config);
+const editAvatarFormValidator = new FormValidator(avatarForm, config);
```

Comment:
* The avatar form is still not validated. See here https://disk.yandex.ru/i/pIshV4xSUB4EKg . Error messages don't appear under the inputs if the form is invalid

Please, check if you have correct params here